### DISCUSSION TITLE
How do you configure grafana dashboards to integrate with StorageGrid dashboards?

### DISCUSSION THREAD

**Discussion by twodotoh** [NetApp/harvest#2638] [Q&A]
I am interested to know how you use Grafana to use use the overview and tenant dashaboards, these seem to require some additional configuration to talk to your storagegrid systems, interested to know where you input this additional configuration?

**Comment by cgrinds** [comment]
hi @twodotoh how did you install Harvest? 

To collect metrics from StorageGRID you need to add new poller, in your harvest.yml config file, that includes the address, username, password of your StorageGRID cluster. Details https://netapp.github.io/harvest/nightly/configure-storagegrid/

**Reply by twodotoh** [reply]
Thank you will take a look.
